Sources/SwiftProtobuf/Message.swift

```diff
@@ -96,15 +117,22 @@ public protocol Message: CustomDebugStringConvertible {
   // JSON encoding/decoding support
   //
 
-  /// Returns a JSON-coded representation of this object as a String.
+  /// Returns a JSON-encoded representation of this object as a `String`.
   func jsonString() throws -> String
```

Comment:
#362 moved `func jsonString()` out of the Message protocol, but it is still part of the public API and still needs to be documented in `JSONTypeAdditions.swift`.